extension/controller/translation/translationWorker.js

```diff
@@ -200,11 +200,15 @@ class TranslationHelper {
               await this.constructTranslationService();
               await this.constructTranslationModel(sourceLanguage, targetLanguage);
               if (withOutboundTranslation) {
-                await this.constructTranslationModel(targetLanguage, sourceLanguage);
-                postMessage([
-                    "displayOutboundTranslation",
-                    null
-                ]);
+                  try {
+                    await this.constructTranslationModel(targetLanguage, sourceLanguage);
+                    postMessage([
+                        "displayOutboundTranslation",
+                        null
+                    ]);
+                  } catch (ex) {
+                      console.warn("Error while constructing a reversed model for outbound translation. It might be not supported.", ex)
```

Comment:
We need to surface this and warn the user instead of swallowing the error. I've added support to that a couple of commits ago, please refer to that and try to reproduce here. It's just basically sending another message to the mediator, one line.